### ISSUE TITLE
Get Khronos to do a RFQ for Maya/Max exporter?

### ISSUE THREAD

**Issue by reduz** [KhronosGroup/glTF#1077]
I noticed they did one for GLTF2 for Blender here:
https://www.khronos.org/rfq/request-for-quote-blenfer-gltf-exporter

Is there any reason something smiilar for Maya/Max does not exist?

**Comment by pjcozzi** [comment]
Ecosystem priorities are being discussed, but this certainty could happen.

The best place to chime in on the ecosystem is #1051.

**Comment by rhokstar** [comment]
@reduz 100% totally agree. At the moment I'm using multiple solutions because Autodesk doesn't support native glTF export. It costs me a lot of time because models, animations, and textures go through many changes over the production cycle.

**Comment by Ziriax** [comment]
At [Wonder Media](http://wondermediagroup.net/), we use an old customized Maya -> OGRE exporter, that we might want to adapt for glTF. Although it contains a lot of custom code to export Maya's TRAX animation clips incrementally, this might be a starting point. Is anyone else working on a Maya to glTF exporter? Otherwise we might give this a shot, since we have a lot of interest in glTF.

**Comment by pjcozzi** [comment]
@Ziriax I am not aware of any current Maya glTF exporters, but the community is very interested in one.  It would be **amazing** if you were to develop this, especially with the head-start you have with the OGRE exporter.

Let me know how the Khronos and the glTF community can help.

:rocket:  :rocket: :rocket:

**Comment by Ziriax** [comment]
@pjcozzi Okay, we'll give it a shot. It seems the Collada -> glTF exporter is a good starting point, it already contains easy to use glTF classes, so in theory it is just a question of replacing the OGRE classes with the glTF classes, but of course, the devil is in the details. For example, the way we deal with Maya's segment scale compensation.

**Comment by pjcozzi** [comment]
Very exciting, keep us posted on your progress!!!

>  It seems the Collada -> glTF exporter...

If you mean https://github.com/KhronosGroup/COLLADA2GLTF/, make sure to check out the glTF 2.0 branch, which is nearly complete but needs validation: https://github.com/KhronosGroup/COLLADA2GLTF/tree/2.0

**Comment by Ziriax** [comment]
I looked at the code yesterday, and it is not as straightforward as I hoped (is it ever?).

So we decided to start from scratch, porting over the code step by step.

We will commit our progress in the develop branch at https://github.com/Ziriax/Maya2glTF

We'll first focus on Windows 10 x64, Visual Studio 2017, and Maya 2018 x64, since that is what we use in-house. 

As soon as the code is usable, I will put in on the master branch.
